### PR TITLE
Admin page UI and components

### PR DESCRIPTION
**Pull request by namanrai**
# Pull Request Description Template

Before making your pull request, these descriptions should be included to allow the reviewer to reference.
<ol>
    <li>Added the UI and the different components for the admin page and linked  it with the login/signup</li>
        <ul>
            <li>Added components for the Profile, Address and Order History Components and general styling for the admin page</li>
        </ul>
    <li>Changes can be found in the components folder under frontend</li>
        <ul>
            <li>merchStore\frontend\src\components\Admin.js</li>
        </ul>
    <li>Some changes are affected to the navigation of the page</li>
        <ul>
            <li>Link it with the User Login and Authentication in future and also access to the user account data</li>
        </ul>
</ol>

### CODE REVIEW

**Review by DROFEL — DISMISSED** [review]
All good!

Can you please look into .gitignore and add .vscode folder. Lets keep repo clean :)

**Review by rickycse — APPROVED** [review]
LGTM

File: frontend/src/components/ProductPopup.js [hunk 1]
```diff
@@ -162,9 +173,18 @@ export default function ProductPopup({ trigger, onClick, product, cart, setCart
 
               <Box></Box>
               <Button
-                sx={button_black}
+                
                 onClick={() => cartHandler()}
                 variant="outlined"
+                sx={{
```
Comment (DROFEL) [inline frontend/src/components/ProductPopup.js]: This one concerns me.

**Review by DROFEL — CHANGES_REQUESTED** [review]
Can you please resolve conflict for product popup before i can approve

**Comment by namanrai** [comment]
Fixed merge conflicts. Please re-review. Thank You

**Review by ShafinMahmudCS — APPROVED** [review]

**Review by rickycse — COMMENTED** [review]
Your ProductPopup.js is behind some changes. Pull the new changes before pushing to the development branch to avoid deleting the Popup changes.

**Comment by namanrai** [comment]
Made the changes and rebased the branch to development.

> Your ProductPopup.js is behind some changes. Pull the new changes before pushing to the development branch to avoid deleting the Popup changes.

**Review by DROFEL — APPROVED** [review]
good

**Pull request merged**